packages/react-component-playground/src/components/welcome-screen.less

```diff
@@ -0,0 +1,20 @@
+.welcome-screen {
+  position: absolute;
+  left: 250px;
+  min-height: 100vh;
+  width: ~"calc(100% - 250px)";
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  &-inner {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
```

Comment:
Sure, but I'll extract those variables into a separate LESS file, I wouldn't want to import the entire `component-playground.less` file into this new component.